android/app/src/main/cpp/audio_mixer.cpp

```
@@ -171,6 +184,22 @@ void AudioMixer::getMixedAudioForDevice(int deviceId, int16_t* outputBuffer, int
     }
 }
 
+void AudioMixer::setDeviceVolume(int deviceId, float volume) {
```

Clang (14.0.6)

[warning] method 'setDeviceVolume' can be made static

(readability-convert-member-functions-to-static)